task/deprecated-image-check/0.1/deprecated-image-check.yaml

```diff
@@ -33,11 +35,18 @@ spec:
       image: registry.access.redhat.com/ubi8/ubi
       script: |
         #!/usr/bin/env bash
-        echo "Querying Pyxis for $(params.IMAGE_REPOSITORY)..."
-        http_code=$(curl -s -k -o $(workspaces.sanity-ws.path)/repository_data.json -w '%{http_code}' "https://catalog.redhat.com/api/containers/v1/repositories/registry/$(params.IMAGE_REGISTRY)/repository/$(params.IMAGE_REPOSITORY)")
+        IFS=$'\n' read -d'' -r -a IMAGE_ARRAY < <(echo -n $BASE_IMAGES_DIGESTS | sed 's/\\n/\'$'\n''/g' && printf '\0')
+        for BASE_IMAGE in ${IMAGE_ARRAY[@]};
+        do
+            IFS=:'/' read -r IMAGE_REGISTRY IMAGE_WITH_TAG <<< "$BASE_IMAGE"; echo "[$IMAGE_REGISTRY] [$IMAGE_WITH_TAG]"
+            IMAGE_REPOSITORY=`echo $IMAGE_WITH_TAG | cut -d ":" -f1`
+            IMAGE_REGISTRY=${IMAGE_REGISTRY//registry.redhat.io/registry.access.redhat.com}
 
-        echo "Response code: $http_code"
-        echo -n $http_code > $(results.PYXIS_HTTP_CODE.path)
+            echo "Querying Pyxis for $(BASE_IMAGE)..."
+            http_code=$(curl -s -k -o $(workspaces.sanity-ws.p"registry.redhat.io"ath)/repository_data.json -w '%{http_code}' "https://catalog.redhat.com/api/containers/v1/repositories/registry/$(IMAGE_REGISTRY)/repository/$(IMAGE_REPOSITORY)")
```

Comment:
`$(workspaces.sanity-ws.p"registry.redhat.io"ath)` looks incorrect, it should be the path of workspace `$(workspaces.sanity-ws.path)`

Reply:
yes. I just noticed it got added while I was testing. will remove it.